iter.go

```diff
@@ -27,43 +28,23 @@ type Iter struct {
 	values     []interface{}
 }
 
-// GetIter returns a new Iter for a given query and its options.
-func GetIter(container ListParamsContainer, query Query) *Iter {
-	var listParams *ListParams
-	formValues := &form.Values{}
-
-	if container != nil {
-		reflectValue := reflect.ValueOf(container)
-
-		// See the comment on Call in stripe.go.
-		if reflectValue.Kind() == reflect.Ptr && !reflectValue.IsNil() {
-			listParams = container.GetListParams()
-			form.AppendTo(formValues, container)
-		}
-	}
-
-	if listParams == nil {
-		listParams = &ListParams{}
-	}
-	iter := &Iter{
-		formValues: formValues,
-		listParams: *listParams,
-		query:      query,
-	}
-
-	iter.getPage()
-
-	return iter
+// Current returns the most recent item
+// visited by a call to Next.
```

Comment:
before I review everything, is this going to the next line too early? Is it because go expects your comment length to be aligned to the function's prototype or something?

Reply:
Yep, definitely —

I left all code + comments exactly the same for now though (they've just been moved around). I want to fix this, but I think I'm going to push it to another PR where it'll be more clear what's changing and a lot easier to read.

Reply:
ah gotcha. I've always been confused at our comment lines length so felt like the right time to ask. Okay going to review the organization for real now.